json-lint/schemas/inventory/adservers/adthrive.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$id": "#AdThrive",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "AdThrive adserver",
+  "description": "https://github.com/ampproject/amphtml/blob/master/ads/adthrive.md",
+  "additionalProperties": false,
+  "required": [
+    "type",
+    "site-id"
```

Comment:
`data-ad-unit` is requiered

Reply:
the problem is that is required for AMP but not for web